### ISSUE TITLE
How to updating this boilerplate?

### ISSUE THREAD

**Issue by borsTiHD** [michalzaq12/electron-nuxt#552]
Hi all,

first of all, thanks for this awesome template.
I'm using it a long time now. But never finaly build an app with it.
My problem is, in the meantime I updated a few dependencies , like vuetify.

My first problem was the missing "utf-8" charset and chars like "ä", "ö", "ü" was unformated after I builded my app.
I found the fix in: 7222627a0004a109218dbf7f89dd37f962c8bc19

My next problem is, that a few components of vuetify aren't building correctly.
The big one I noticed is the `v-main` component.
Left side after build and right side in developement enviroment: [screenshot](https://i.postimg.cc/ht3MdyyP/build-vuetify.png)

So I think my problem is, that my dependencies are just too old for the latest changes.
My question is, how can I securly update everything (or just critical dependencies)?
Is there a way to update the whole template, with or without loosing my own customized code?

Thanks in advanced. :)

**Comment by dealenx** [comment]
Hi,

Your issue is thought-provoking, I am also interested how to update properly template in my applications in the future.

if you will have a solution or thoughts on how to solve this problem, please share here.

**Comment by borsTiHD** [comment]
Hi,

in the meanwhile, I updated every "boilerplate related" script manually.
But it didn't solve my problem with the missing `v-main` component.

The missing `v-main` component is in my template `default.vue` file.
Again, if I run my app in dev mode (`yarn run dev`) the component is imported correctly.
If I build my app with (`yarn run build`) the `v-main` component is not imported.

But I found a workaround to solve my problem if anyone else got the same problem.
I imported the vuetify component on my own and registered it:

`default.vue`:
```
<template>
    <v-app>
        <v-main>
            <v-container>
                <nuxt />
            </v-container>
        </v-main>
    </v-app>
</template>

<script>
import { VMain } from 'vuetify/lib'

export default {
    name: 'App',
    components: {
        VMain
    },
    [...]
}
</script>
```
